README.md

```diff
@@ -10,6 +10,12 @@ with the rich connectors in Flink and handle the failures.
 
 Currently, Deep Learning on Flink supports TensorFlow and PyTorch.
 
+## Supported Operating System
+Deep Learning on Flink is tested and supported on the following 64-bit systems:
+
+- Ubuntu 18.04 or later
```

Comment:
Given that we mention `..tested and supported...` above, we should probably only mention the version that is actually tested.

How about we remove the `or later` here?

Reply:
It is updated at e87d171